### DISCUSSION TITLE
Refactored the entire ansible project to pass ansible-lint: production

### DISCUSSION THREAD

**Discussion by shaneholloman** [vmware/packer-examples-for-vsphere#932] [Ideas]
Is that something you'd be interested to use. Its bit of a pain point for me when ya'lls repo is updated, so I rather share it, if you want it ...

**Comment by shaneholloman** [comment]
- fully qualified module names
- properly formed blocks
    - named blocks
    - corrected indentation
    - moved the when directive to top of blocks
- idempotent commands
- prefixed task_names var with role name
- added some changed when's to silence linter where needed
- suse play had some hard coded values, so ripped those out to var/main.yml

**Reply by tenthirtyam** [reply]
sure!!

**Comment by shaneholloman** [comment]
I dont understand how CLAs work, so Im not interested in signing one for a pull request. Happy to zip up the ansible dir and send it you ...

**Reply by tenthirtyam** [reply]
please feel free:

**Comment by shaneholloman** [comment]
is it true to say that the packer and ansible var with the same name of `enable_cloudinit` are dependant on each other. reason for asking is if I change it in ansible to pass lint rules I with have search and replace in the packer confs as well well?

**Reply by tenthirtyam** [reply]
No, it only needs to be changed in the argument that the ansible provisioner is using. 

But instead, I've taken the provided content and refactored it a little more to move this to the linux-playbook.yml as a var since it's used by base and configure.

**Comment by shaneholloman** [comment]
after updating ansible and ansible-lint just now I have found more fixes.
I should wait for what you've cooked up firstly though ...

**Reply by tenthirtyam** [reply]
Tracked under https://github.com/vmware-samples/packer-examples-for-vsphere/pull/934.
